test/Microsoft.ML.TorchSharp.Tests/Microsoft.ML.TorchSharp.Tests.csproj

```diff
@@ -0,0 +1,40 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <AssemblyName>Microsoft.ML.TorchSharp.Tests</AssemblyName>
+    <StrongNameKeyId>Test</StrongNameKeyId>
+    <NoWarn>$(NoWarn)</NoWarn>
```

Comment:
><NoWarn>$(NoWarn)</NoWarn> [](http://example.com/codeflow?start=4&length=26)

This looks weird to me :-)

Reply:
Yeah... was there in the project I copied this from so it came along. I have now removed all of them.